docs/pages/toolchain/katana/genesis.md

```diff
@@ -63,16 +63,16 @@ The genesis config file is a JSON file that contains the following fields:
 - `contracts` _Genesis contract deployments._
   - <CONTRACT_ADDRESS> _The address of the contract._
     - `class`  
-      _The class of the contract._
+      _The name of the class._
     - `balance` (optional)  
       _The balance allocated to the contract._
     - `storage` (optional)  
       _Key-value pairs for the contract's storage._
 - `classes` _Classes to declare at genesis._
   - `class`  
     _The path to the class artifact file relative to the genesis config file, or the full class artifact object._
-  - `classHash` (optional)
-    _The hash of the class. To override the actual class hash that will be computed from the class definition itself._
+  - `name` (optional)
+    _The name of the class._
```

Comment:
we still need both. we still allow specifying the exact class hash, this is for overriding the class hash value which otherwise would be computed from the artifact directly. 

at the same time, we also have a 'helper' field where user can provide a name to the class entry. The name can then be used when specifying the `class` field of a contract configuration (eg the `class` field in the `feeToken` config or the `class` field in the `universalDeployer` config).

you can refer to this https://github.com/dojoengine/dojo/blob/6ae661d4e272c0c0920858e255a5fd89aa40ad53/crates/katana/primitives/src/genesis/json.rs#L93-L101